jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebDescriptor.java

```diff
@@ -87,156 +86,13 @@ protected InputSource resolveEntity(String pid, String sid)
             {
                 if (!mapped)
                 {
-                    mapResources();
+                    URL url = WebDescriptor.class.getResource("catalog.xml");
```

Comment:
should this be called `catalog-ee10.xml` for clarity?

Reply:
It can be any name we want it to be.
It's in the `org/eclipse/jetty/ee10/webapp/` directory already, does it really need to be called `catalog-ee10.xml` ?

Reply:
I assume we will end up with an ee9 and ee8 version, so keeping the names different would probably be good... but not absolutely necessary.